Library/Homebrew/sorbet/rbi/gems/activesupport@6.1.7.2.rbi

```diff
@@ -3415,56 +3415,6 @@ class Regexp::Token < ::Struct
   end
 end
 
-class String
```

Comment:
This serves as a type guard to avoid introducing monkey-patch dependencies into typed code while the deletion of `require "active_support/core_ext/string/inflections"` in global.rb is pending